apps/code-examples/src/app/code-examples/phone-field/phone-field/basic/demo.component.html

```diff
@@ -1,7 +1,22 @@
-<form class="phone-field-demo" [formGroup]="phoneForm">
-  <sky-input-box labelText="Phone number">
-    <sky-phone-field [defaultCountry]="'us'">
-      <input formControlName="phoneControl" skyPhoneFieldInput />
-    </sky-phone-field>
-  </sky-input-box>
-</form>
+<div class="sky-padding-even-lg">
+  <form class="phone-field-demo" [formGroup]="phoneForm">
+    <sky-input-box
+      helpPopoverTitle="How we use your phone number"
+      hintText="Enter the phone number where you can be reached before 5 p.m., Monday through Friday."
+      labelText="Phone number"
+      [helpPopoverContent]="inlineHelpPopover"
+    >
+      <sky-phone-field [defaultCountry]="'us'">
+        <input formControlName="phoneControl" skyPhoneFieldInput />
+      </sky-phone-field>
+    </sky-input-box>
```

Comment:
do we want to demo how to add custom form errors in input box here too? or just have that in the input box code example?